0200-background.Rmd

```diff
@@ -1,22 +1,18 @@
-
+Wet'suwet'en hereditary territory covers an area of 22,000km^2^ including the Bulkley River watershed, Zymoetz River watershed, Morice River watershed and portions of the Nechako River watershed. The Wet'suwet'en people are a matrilineal society organized into the Gilseyhu (Big Frog), Laksilyu (Small Frog), Tsayu (Beaver clan), Gitdumden (Wolf/Bear) and Laksamshu (Fireweed) clans. Within each of the clans there are are a number of kin-based groups known as Yikhs or House groups. The Yikh is a partnership between the people and the territory. Thirteen Yikhs with Hereditary Chiefs manage a total of 38 distinct territories upon which they have jurisdiction. Within a clan, the head Chief is entrusted with the stewardship of the House territory to ensure the Land is managed in a sustainable manner. Inuk Nu'at'en (Wet'suwet'en law) governing the harvesting of fish within their lands are based on values founded on thousands of years of social, subsistence and environmental dynamics. The Yintahk (Land) is the centre of life as well as culture and it's management is intended to provide security for sustaining salmon, wildlife, and natural foods to ensure the health and well-being of the Wet'suwet'en [@officeofthewetsuweten2013Wetsuwet; @OfficeoftheWetsuweten2021; @flnrord2017NaturalResource]. Witset is a federally recognized First Nations band of the Wet'suwet'en people and is located between Smithers and Hazelton, on the west side of the Bulkley River. Previously known as Moricetown, this nation is home to approximately 1790 people. This small community was built in the early 1900s but has served as the Wet'suwet'en fishing grounds for thousands of years [@witsetf2019].
```

Comment:
there are 6 Wet'suwet'en bands so we should list them all.  https://en.wikipedia.org/wiki/Wet%CA%BCsuwet%CA%BCen  Probably best to just note that they all exist rather than writing a paragraph about each